content/en/docs/how-tos/interact-with-running-jobs.md

```diff
@@ -286,3 +286,197 @@ $ openshift-tests run
 ```
 
 See the [openshift-tests README](https://github.com/openshift/origin/blob/master/test/extended/README.md) for more information.
+
+## How To Run Test Suites using cluster-bot Development Cluster
+
+There are times when you want to run the `openshift-tests` binary to test or debug a new test without having
+to use Prow/CI.  To achieve this goal, we use the logs from Prow jobs to find out how it runs suites of tests,
+set up minimal local environment, and then run the `openshift-tests` binary locally.  The following sections
+show how to do this.
+
+### Looking at Prow Logs For `openshift-tests`
+
+Prow jobs that run suites of tests using `openshift-test` produce a log called `build-log.txt` that shows
+exactly how the `openshift-tests` binary was executed.  We can use this information to determine how to
+run the test suites against a development cluster (e.g., one created by cluster-bot).
+
+The `build-log.txt` file usually resides in a path similar to:
+
+```bash
+https://<GCSPrefix>/logs/<jobName>/<jobID>/artifacts/<e2eJob>/openshift-e2e-test/build-log.txt
+```
+
+In that log, you can see how `openshift-tests` was run.  For example:
+
+```bash
+openshift-tests run openshift/conformance/serial
+                    --provider '{"type":"aws",...}'
+                    -o /logs/artifacts/e2e.log
+                    --junit-dir /logs/artifacts/junit
+```
+
+Given how prow runs `openshift-tests`, we will mimic this except run it manually.
+
+When `openshift-tests` is run in prow, the paths for `-o` and `--junit-dir` lead to local directories
+residing in a pod where `openshift-tests` is run.  Those contents are then copied to a GCS bucket and
+then end up in a location accessible from a link as shown above. When you look at artifacts in a prow
+job, the links will look similar to this:
+
+```bash
+https://<GCSPrefix>/logs/<jobName>/<jobID>/artifacts/<e2eJob>/openshift-e2e-test/artifacts/e2e.log
+https://<GCSPrefix>/logs/<jobName>/<jobID>/artifacts/<e2eJob>/openshift-e2e-test/artifacts/junit/*
+```
+
+For running `openshift-tests` locally, we will create directories on our local disk and maintain the same
+directory structure and relative placement of files; we will also set `--provider`, to
+`{"type":"<aCloudProvider>"}` where `<aCloudProvider>` is one of `aws` or `gcp`.
+
+NOTE: For `azure`, running `openshift-tests` is more complicated due to the need for more credentials and is not covered
+in this section.
```

Comment:
Nice approach to figuring this out.

I'd be tempted to cut this section above, it would keep the docs punchy for the developer looking for how do I run openshift-tests, get them just the info they need as quick as possible. It might help someone reverse engineer how to do it if something's changed and the docs are out of date, but by that point we've already failed pretty good and they're probably going to give up or come ask on slack. The note on Azure could be moved lower if we do remove.

Might be good to dump it into the Jira though to document how you figured this out.

Let me know if you disagree though.

Reply:
Makes sense.  I added https://issues.redhat.com/browse/TRT-356 for investigating azure.

I'm hoping my section on "Looking at Prow Logs For openshift-tests" helps people figure out exactly how `openshift-tests` is run, when they read this doc, in case this doc becomes out of date.